components/actions/ActionCardList.js

```diff
@@ -102,14 +103,20 @@ function ActionCardList(props) {
   const { actions, groupBy, headingHierarchyDepth, includeRelatedPlans } =
     props;
   const theme = useTheme();
-
-  const groups = groupActions(groupBy, headingHierarchyDepth, actions, theme);
+  const { t } = useTranslation();
+  const groups = groupActions(
+    groupBy,
+    headingHierarchyDepth,
+    actions,
+    theme,
+    t
+  );
 
   return (
     <ActionsList>
       {groups.map((group) => (
         <ActionGroup key={group.id} tag="li">
-          {groups.length > 1 && (
+          {(groups.length > 1 || group.id === 'zzzz') && (
```

Comment:
We could store this as a variable e.g. `OTHER_GROUP_ID` and reference it here and on lines 90 & 95. That should make the code a bit more readable and future devs won't just think this group is sleeping 💤